app/views/lois/show.html.erb

```diff
@@ -1,28 +1,113 @@
-<h1> LOI SHOW </h1>
-
-<p> Name: <%= @loi.name %> </p>
-<p> Email: <%= @loi.email %> </p>
-
-<ol>
-  <% @sections.each do |section| %>
-    
-    <h3><li><%= section.name %></li></h3>
-      <ul>
-        <% @loi.answers.each do |answer| %>
-
-          <% if answer.question.section_id == section.id %>
-            <li><p> <%= answer.question.question %>: <%= answer.answer %> </p></li>
-          <% end %>
-        <% end %>
-      </ul>
-  <% end %>
-</ol>
-
-<% if user_signed_in? %>
-  <%= link_to "Rate", "/ratings/new?loi=#{@loi.id}", :class => "btn btn-default", :role => "button" %>
-<% end %>
-
-<% if current_user && current_user.admin %>
-  <%= link_to "Edit", "/lois/#{@loi.id}/edit/", :class => "btn btn-default", :role => "button" %>
-  <%= link_to "Delete", "/lois/#{@loi.id}", method: :delete, data:{confirm: "Are You Sure"}, :class => "btn btn-default", :role => "button" %>
-<% end %>
+<div class="form-heading">
+
+  <h1> LOI SHOW </h1>
+
+  <div class="form">
+
+    <p> Name: <%= @loi.name %> </p>
+    <p> Email: <%= @loi.email %> </p>
+
+    <ol>
+      <% @sections.each do |section| %>
+
+        <h3><li><%= section.name %></li></h3>
+
+          <ul>
+
+            <% if section.name == "Organizational Capacity Challenges" %>
+                <table class="table">
+                  <tr>
+                    <th>Focus Area</th>
+                    <th>Please provide a short description of your top 1-3 challenge(s) in each area that applies</th>
+                    <th>Priority Rank</th>
+                    <th>2015 FTEs</th>
+                    <th>2016 FTEs</th>
+                    <th>2017 FTEs</th>
+                  </tr> 
+            <% elsif section.name == "Demographics" %>
+                <table class="table">
+                  <tr>
+                    <th>Sector</th>
+                    <th>Mark an “x” if you have programs addressing this sector</th>
+                  </tr>
+            <% elsif section.name == "Geographic Focus" %>
+                <table class="table">
```

Comment:
@heydanhey  This isn't absolutely necessary, but let's go ahead and fix this now. To shorten the amount of code, you can have `<table class="table">` and then begin the code with the if statements and the tr>th tags. That way you could also get rid of the `elsif` and `</table>` below. I think that would get rid of 6 lines of code or so. You can do this with the lois/new form above as well.